cmd/px-resource-gateway/px_resource_gateway.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	_ "net/http/pprof"
+	"os"
+
+	"github.com/libopenstorage/grpc-framework/pkg/util"
+	grpcServer "github.com/libopenstorage/grpc-framework/server"
+	server "github.com/libopenstorage/operator/pkg/px-resource-gateway"
+	"github.com/libopenstorage/operator/pkg/version"
+	pb "github.com/libopenstorage/operator/proto"
+	"github.com/sirupsen/logrus"
+	log "github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+	"google.golang.org/grpc"
+)
+
+const (
+	socket = "/tmp/px-resource-gate.sock"
+)
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "openstorage-operator"
+	app.Usage = "Operator to manage openstorage clusters"
+	app.Version = version.Version
+	app.Action = run
+
+	app.Flags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "verbose",
+			Usage: "Enable verbose logging",
+		},
+	}
+
+	if err := app.Run(os.Args); err != nil {
+		log.Fatalf("Error starting openstorage operator: %v", err)
+	}
+}
+
+func run(c *cli.Context) {
+	logrus.SetLevel(logrus.DebugLevel)
+	semaphoreServer := server.NewServer()
+
+	// TODO: add security
+	config := &grpcServer.ServerConfig{
+		Name:         "px-resource-gate",
+		Address:      "127.0.0.1:9009",
```

Comment:
I'm not sure if this will work. Listening on 127.0.0.1 will force it to not accept connections from any other system and only from the local system. You want `:9009` instead.